Jenkinsfile

```diff
@@ -6,7 +6,7 @@ pipeline {
   }
   environment {
     stagingPostgresDbUrl = credentials('STAGINGMETADATAWSDBURL')
-    fallBackPostgresDbUrl = credentials('FALLBACKMETADATAWSDBURL')
+    fallBackPostgresDbUrl = credentials('PRODMETADATAWSDBURL')
```

Comment:
I think we need a discussion on how we want to deploy the DBs.

Reply:
So until we get a production postgres and proxy corrected we gonna use same database for fallback and prod

Reply:
OK! Once we have production instances set up and properly configured for cross instance replication we can use different databases for fallback and production. For now, we have to use the same database.